sentry_js/static/src/js/web_client.js

```diff
@@ -0,0 +1,27 @@
+odoo.define('sentry_js.web_client', function (require) {
+    'use strict';
+
+    var WebClient = require('web.WebClient');
+
+    WebClient.include({
```

Comment:
Is it possible that this could cause race conditions? If I use WebClient and do not depend on your module (satisfies all the core modules) then it will be 50/50 case where the `bind_events` is modified with sentry and not, no?

Recently just had a similar case.

Reply:
I really don't know, all I can say it's done [similarly](https://github.com/odoo/odoo/blob/e2532f855f7088ea772903a610e883c7479fbf36/addons/calendar/static/src/js/base_calendar.js#L243-L256) in Odoo core. If you know a way to possibly avoid it, please share.

Reply:
nah, don't know the reason why it should work in Odoo as well. Maybe `session.is_bound` has something to do with it.